src/compas/geometry/_core/angles.py

```diff
@@ -155,6 +155,9 @@ def angle_vectors_projected(u, v, normal, deg=False, tol=None):
     u_cross = cross_vectors(u, normal)
     v_cross = cross_vectors(v, normal)
 
+    if TOL.is_allclose(u_cross, [0.0, 0.0, 0.0]) or TOL.is_allclose(v_cross, [0.0, 0.0, 0.0]):
+        raise ValueError("Cannot compute angle between vectors projected onto a plane defined by the normal vector. One of the vectors is parallel to the normal vector.")
```

Comment:
I think the right place to put this is https://github.com/compas-dev/compas/blob/ba669cb9373370a82e9fd99e301a2dfbc009d884/src/compas/geometry/_core/angles.py#L47-L49

however, that's potentially a breaking change. since `angle_vectors_projected` is relatively new and specialized, perhaps we keep it here for now. @tomvanmele any objections to merging this?